### ISSUE TITLE
Phy WaveformView

### ISSUE THREAD

**Issue by justindyao** [cortex-lab/KiloSort#216]
Hi KiloSort team,

I'm very new to kilosort and just started using Phy to check my data. Currently, WaveformView is displaying "high_pass" versions of the waveforms (waveforms, mean_waveforms, and templates). Below are screenshots of examples.

<img width="1439" alt="Screen Shot 2020-02-05 at 9 35 00 PM" src="https://user-images.githubusercontent.com/60723439/73901759-bbeb2980-4861-11ea-86b9-f687b1716832.png">
<img width="1440" alt="Screen Shot 2020-02-05 at 9 34 41 PM" src="https://user-images.githubusercontent.com/60723439/73901760-bc83c000-4861-11ea-9cc6-976ecf6247fd.png">
<img width="1440" alt="Screen Shot 2020-02-05 at 9 34 21 PM" src="https://user-images.githubusercontent.com/60723439/73901763-bd1c5680-4861-11ea-817e-a506282c93a7.png">

Is there a way to change the display so that it is not plotting the high_pass versions?

Many thanks in advance,
Justin

**Comment by rossant** [comment]
Yes press alt r


Le jeu. 6 févr. 2020 à 03:54, justindyao <notifications@github.com> a
écrit :

**Comment by justindyao** [comment]
Thanks. That changed the display text but did not change the waveforms themselves.

**Comment by rossant** [comment]
even if you press it repeatedly? 
Can you also update phy and phylib to the latest versions
PS: this issue should actually be on the phy repository

**Comment by justindyao** [comment]
Yes, the waveforms still have not updated even if I press alt r repeatedly and after I have updated phy and phylib my terminal lists: 
"actions:204 Error when executing action Switch raw data filter."

it is accompanied with this message:
"FutureWarning, /opt/anaconda3/envs/phy2/lib/python3.7/site-packages/dask/array/core.py:1333: FutureWarning: The `numpy.quantile` function is not implemented by Dask array. You may want to use the da.map_blocks function or something similar to silence this warning. Your code may stop working in a future release." Not sure if they are related.

I tested this out on another computer with an older version and things seem to work appropriately.

Thanks for the help.

Justin

**Comment by rossant** [comment]
There was something wrong in the phylib repository. Could you update phylib (master branch) and try again? Sorry for the inconvenience.

**Comment by justindyao** [comment]
thanks, it works now!
